Header header
bool is_left
bool is_right
bool resume_image
double lane_correct
bool stop
